website_and_docs/content/documentation/grid/configuration/cli_options.en.md

```diff
@@ -349,3 +350,50 @@ driver.quit();
 ```
 
 Set the custom capability to `false` in order to match the Node B.
+
+#### Specifying path from where downloaded files can be retrieved
+
+At times a test may need to access files that were downloaded by it on the Node. To retrieve such files, following can be done.
+
+##### Start the Hub
+```
+java -jar selenium-server-<version>.jar hub
+```
+
+##### Start the Node with downloads path specified
+```
+java -jar selenium-server-<version>.jar node --downloads-path /usr/downloads
+```
+
+##### Sample that retrieves the downloaded file
+
+```java
+// The file can be downloaded by accessing
+// curl -X GET "http://localhost:4444/session/<sessionId>/se/file?filename=my_file.pdf"
+
+HttpRequest request = new HttpRequest(
+    HttpMethod.GET,
+    String.format("/session/%s/se/file", driver.getSessionId()));
+request.addQueryParameter("filename", "my_appointments-1.pdf");
+// Here gridUrl is http://localhost:4444
+try (HttpClient client = HttpClient.Factory.createDefault().createClient(gridUrl)) {
+  HttpResponse response = client.execute(request);
+  Map<String, Object> map = new Json().toType(string(response), Json.MAP_TYPE);
+  // The returned map would contain 2 keys viz.,
+  // filename - This represents the name of the file (same as what was provided by the test)
+  // contents - Base64 encoded String which contains the zipped file.
+  String encodedContents = map.get("contents").toString();
+
+  //The file contents would always be a zip file and has to be unzipped.
+  Zip.unzip(encodedContents, dirToCopyTo);
+}
```

Comment:
This code example is incomplete. We should either provide a complete one (this means all variables are declared and with a value, so the user can copy & paste the whole code), or maybe just leave the `curl` example.